doc/classes/ColorPicker.xml

```diff
@@ -168,9 +168,6 @@
 		<theme_item name="color_hue" data_type="icon" type="Texture2D">
 			Custom texture for the hue selection slider on the right.
 		</theme_item>
-		<theme_item name="color_okhsl_hue" data_type="icon" type="Texture2D">
-			Custom texture for the H slider in the OKHSL color mode.
-		</theme_item>
```

Comment:
This technically breaks compatibility with 4.3, but I suppose this is a really niche configuration option and we might expect that nobody is using it.

If some do, we'll have to see and possibly reconsider.